test/org/github/jamm/MemoryMeterTest.java

```diff
@@ -424,23 +419,23 @@ public void addChildrenTo(MeasurementStack stack) {
         }
     }
 
-//    @Test
-//    public void testMeasureDeepString() {
-//
-//        String[] strings = new String[] {null,
-//                                         "",
-//                                         "a",
-//                                         "a bit longuer",
-//                                         "significantly longuer",
-//                                         "...... really ...... really .... really ... really .... longuer",
-//                                         "with a chinese character: 我"};
-//
-//        MemoryMeter reference = MemoryMeter.builder().withGuessing(Guess.INSTRUMENTATION).build();
-//        MemoryMeter meter = MemoryMeter.builder().withGuessing(guess).build();
-//
-//        for (String string : strings) {
-//            assertEquals(reference.measureDeep(string), meter.measureDeep(string));
-//            assertEquals(reference.measureDeep(string), meter.measureStringDeep(string));
-//        }
-//     }
+    @Test
```

Comment:
It is the opposite. Things were wrong unless we added special characters. LATIN1 characters can fit on 1 byte so if the JVM detect that a String has only latin 1 characters it will compress the data and use half of the bytes it used with Java 8.